cypress-tests/cypress/e2e/editor/data-source/snowflakeHappyPath.cy.js

```diff
@@ -118,7 +118,7 @@ describe("Data sources", () => {
     );
   });
 
-  it("Should verify the functionality of PostgreSQL connection form.", () => {
+  it.skip("Should verify the functionality of PostgreSQL connection form.", () => {
```

Comment:
Is it fine to skip? Also the spec text should be for snowflake right?

Reply:
Yes, Due to credentials are expired we are going to skip this.
I have updated test case name.